### PR TITLE
Newlines are no longer allowed in the title field.

### LINKED ISSUE #7106: Aztec: Post title should not be multiline
### Expected behavior
When entering text into the post title the next button goes to the post content section.

### Actual behavior
When entering text into the post title the next button acts like a return button and adds newlines.

### Steps to reproduce the behavior
1. Tap in the title field to give it focus.
2. Tap the quote toolbar button.
3. Result: A blockquote is started in the post content area.

img_8419

##### Tested on unknown, iOS 10, WPiOS 7.5 beta

Internal reference: `p4a5px-1J0-p2`

### PR DESCRIPTION
**Pull request by diegoreymendez**
### Description

Fixes #7106 

When enter is pressed while the caret is in the title view, the caret will be moved to the content view.

If text containing newlines is pasted into the title view, the newlines will be removed before pasting.

### Testing:

**Test 1:**

* Open the editor.
* Place the caret in the title view and press enter.
* The caret should be moved to the first position of the rich text view.

PS: Combine the above steps with having content in the title and rich text view.

**Test 2:**

* Copy text containing several newlines.
* Pase the text in the title view.
* Make sure all newlines are removed.

### CODE REVIEW

**Comment by diegoreymendez** [comment]
I'm adding @SergioEstevao and @jleandroperez as reviewers too, so whoever is free first can tackle this.

File: WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift [hunk 1]
```diff
@@ -1111,6 +1122,61 @@ extension AztecPostViewController : UITextViewDelegate {
...
+    /// This method performs all necessary checks to verify if the title text can be changed,
+    /// or if some other action should be performed instead.
+    ///
+    /// - Important: this method sanitizes newlines, since they're not allowed in the title.
+    ///
+    /// - Parameters:
+    ///     - range: the range that would be modified.
+    ///     - text: the new text for the specified range.
+    ///
+    /// - Returns: `true` if the moficiation can take place, `false` otherwise.
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift]: Typo: modification

**Comment by diegoreymendez** [comment]
@frosty - I'll replace them with newlines, since it'd probably look better.  Thanks!

**Comment by frosty** [comment]
@diegoreymendez You mean with spaces? Please don't replace the newlines with newlines 😁

**Comment by diegoreymendez** [comment]
@frosty - I've addressed your comments.

**Comment by diegoreymendez** [comment]
Thank you @frosty and @jleandroperez !

**Pull request merged**